src/test/resources/storage-loan-6-2.json

```diff
@@ -21,6 +21,11 @@
       "description": "ID of the item lent to the patron",
       "type": "string"
     },
+    "itemEffectiveLocationAtCheckOut": {
```

Comment:
This property was introduced for `loan-storage 6.3`, rather than retrofitting it to the copy of the 6.2 schema, please copy the 6.3 schema here and change the validation to use it.

I'm aware this process is confusing and unfamiliar, it is rather a compromise and fairly new. Please refer to this [documentation](https://github.com/folio-org/mod-circulation/blob/master/doc/automated-testing.md#schema-validation), and share feedback if it doesn't make sense.